### PR TITLE
Rename fixtures.py to conftest.py

### PR DESCRIPTION
**Pull request by ebenahar**
See the discussion about this here - https://github.com/red-hat-storage/ocs-ci/issues/200

### CODE REVIEW

File: tests/manage/test_pvc_deletion_during_io.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from ocs import defaults, exceptions, constants
 from resources import pod
 from ocsci.testlib import ManageTest, tier1
-from tests.fixtures import (
+from tests.conftest import (
```
Comment (andrewschoen) [inline tests/manage/test_pvc_deletion_during_io.py]: You shouldn't need to import fixtures. The ``@pytest.mark.usefixtures`` call below could just pass the name of the fixture as a string. For example:

```
@pytest.mark.usefixtures(
    'create_rbd_secret',
    'create_ceph_block_pool',
    ...
)
```

Here's a link to the docs with more info on that: https://docs.pytest.org/en/latest/fixture.html#using-fixtures-from-classes-modules-or-projects

Comment (mbukatov) [reply tests/manage/test_pvc_deletion_during_io.py]: As @andrewschoen already noted, this is not expected way to use fixtures.

**Comment by mbukatov** [comment]
See https://github.com/mbukatov/ocs-ci/commit/345329544500843c51b32dc56265084d1fb0c74c

File: tests/manage/test_pvc_deletion_during_io.py (hunk 1 above)
Comment (ebenahar) [reply tests/manage/test_pvc_deletion_during_io.py]: Great input guys!
Will fix accordingly

**Comment by ebenahar** [comment]
tests/manage/test_pvc_deletion_during_io.py::TestCaseOCS371::test_run_io_and_delete_pvc 
------------------------------------------------------------------------------------------------- live log setup --------------------------------------------------------------------------------------------------
23:33:33 - MainThread - root - INFO - All logs are located under /tmp/logs
23:33:33 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
23:33:40 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-p7l2b ceph auth get-key client.admin --format json-pretty
23:33:53 - MainThread - resources.ocs - INFO - Adding Secret with name secret-test-1323335316
23:33:53 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig create -f /tmp/Secretqqqx29ng -o yaml
23:33:57 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Secret secret-test-1323335316 -o yaml
23:34:01 - MainThread - resources.ocs - INFO - Adding CephBlockPool with name cbp-test-1323340167
23:34:01 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig create -f /tmp/CephBlockPooliqevu7ev -o yaml
23:34:06 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get CephBlockPool cbp-test-1323340167 -o yaml
23:34:11 - MainThread - tests.helpers - INFO - Verifying that block pool cbp-test-1323340167 exists
23:34:11 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Pod --selector=app=rook-ceph-tools -o yaml
23:34:15 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig rsh rook-ceph-tools-76c7d559b6-p7l2b ceph osd lspools --format json-pretty
23:34:29 - MainThread - resources.ocs - INFO - Adding StorageClass with name storageclass-test-1323342998
23:34:29 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig create -f /tmp/StorageClassfc02j0gt -o yaml
23:34:33 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get StorageClass storageclass-test-1323342998 -o yaml
23:34:39 - MainThread - resources.ocs - INFO - Adding PersistentVolumeClaim with name pvc-test-1323343933
23:34:39 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig create -f /tmp/PersistentVolumeClaim4dg4g6ka -o yaml
23:34:43 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get PersistentVolumeClaim pvc-test-1323343933 -o yaml
23:34:46 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get PersistentVolumeClaim pvc-test-1323343933 -o yaml
23:34:50 - MainThread - resources.ocs - INFO - Adding Pod with name pod-test-1323345028
23:34:50 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig create -f /tmp/POD_7btig6sr -o yaml
23:34:55 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Pod pod-test-1323345028 -o yaml
23:35:03 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Pod pod-test-1323345028 -o yaml
23:35:09 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Pod pod-test-1323345028 -o yaml
-------------------------------------------------------------------------------------------------- live log call --------------------------------------------------------------------------------------------------
23:35:20 - MainThread - resources.pod - INFO - Running I/O on pod pod-test-1323345028
23:35:20 - Thread-1 - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig rsh pod-test-1323345028 bash -c "let i=0; while true; do echo Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. >> /var/lib/www/html/test$i; let i++; sleep 0.01; done"
23:35:22 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig rsh pod-test-1323345028 bash -c "find /var/lib/www/html/test1"
23:35:30 - MainThread - tests.manage.test_pvc_deletion_during_io - INFO - Deleting PVC pvc-test-1323343933
23:35:30 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig delete PersistentVolumeClaim pvc-test-1323343933 --wait=false
23:35:31 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get PersistentVolumeClaim pvc-test-1323343933
23:35:49 - MainThread - tests.manage.test_pvc_deletion_during_io - INFO - Deleting pod pod-test-1323345028
23:35:49 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig delete Pod pod-test-1323345028
23:35:51 - Thread-1 - utility.utils - ERROR - CMD error:: command terminated with exit code 137

23:35:51 - Thread-1 - resources.pod - INFO - I/O command got terminated as expected
23:35:54 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get PersistentVolumeClaim pvc-test-1323343933
23:35:56 - MainThread - utility.utils - ERROR - CMD error:: Error from server (NotFound): persistentvolumeclaims "pvc-test-1323343933" not found

PASSED
------------------------------------------------------------------------------------------------ live log teardown ------------------------------------------------------------------------------------------------
23:35:56 - MainThread - utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get StorageClass storageclass-test-1323342998 -o yaml
23:36:05 - MainThread - utility.utils - INFO - Executing command: oc --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig delete StorageClass storageclass-test-1323342998
23:36:10 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get CephBlockPool cbp-test-1323340167 -o yaml
23:36:14 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig delete CephBlockPool cbp-test-1323340167
23:36:19 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig get Secret secret-test-1323335316 -o yaml
23:36:23 - MainThread - utility.utils - INFO - Executing command: oc -n openshift-storage --kubeconfig /home/ebenahar/ebenahar-ocs-dir/auth/kubeconfig delete Secret secret-test-1323335316


=========================================================================================== 1 passed in 175.11 seconds ============================================================================================

**Comment by vasukulkarni** [comment]
@ebenahar there is a conflict that you have to resolve in your branch.

**Comment by clacroix12** [comment]
We can't rename `fixtures.py` to `conftest.py` as we already have a `conftest.py`. We will need to move the fixtures from `fixtures.py` into the existing `conftest.py`.

**Comment by vasukulkarni** [comment]
@ebenahar needs rebase and addition to existing fixture file.

**Comment by vasukulkarni** [comment]
@ebenahar ping?

**Comment by vasukulkarni** [comment]
@ebenahar do you mind, If I take this up?

**Comment by ebenahar** [comment]
Closing for now as conftest.py already exists

**Pull request closed without merging**